test/COWShedProxy.t.sol

```diff
@@ -53,11 +54,47 @@ contract COWShedProxyTest is BaseTest {
         vm.expectRevert(COWShedProxy.InvalidInitialization.selector);
         COWShed(payable(address(proxy))).trustedExecuteHooks(new Call[](0));
 
-        COWShed(payable(address(proxy))).initialize(address(factory));
+        COWShed(payable(address(proxy))).initialize(address(factory), true);
         assertImpl(address(proxy), address(cowshed));
 
         // shouldnt initialize again
         vm.expectRevert(COWShed.AlreadyInitialized.selector);
-        COWShed(payable(address(proxy))).initialize(address(factory));
+        COWShed(payable(address(proxy))).initialize(address(factory), true);
+    }
+
+    function testProxyClaimWithResolver() external {
+        COWShed cowshed = new COWShed();
+
+        // should not be able to use proxy before initialization
+        COWShedProxy proxy = new COWShedProxy(address(cowshed), user.addr);
+        assertImpl(address(proxy), address(cowshed));
```

Comment:
nit: adjust comments to reflect missing `vm.expectRevert` / `trustedExecuteHooks` - or fix copy/pasta